lib/api/controllers/routerController.js

```diff
@@ -54,7 +54,7 @@ module.exports = function RouterController (kuzzle) {
       executeFromRest.call(kuzzle, params, request, response);
     });
 
-    api.get('/_getStats', function (request, response) {
+    api.post('/_getStats', function (request, response) {
```

Comment:
also OK for a "1970-01-01" default value for starTime... just make it clear in the doc ;)

so.... finally, getAllStats would be an alias for getStats with no parameters ?